test/e2e/components/vl-link.js

```diff
@@ -1,9 +1,14 @@
 const { VlElement } = require('vl-ui-core').Test;
+const { By } = require('vl-ui-core').Test.Setup;
+const { VlIcon } = require('vl-ui-icon').Test;
 
-class VlLink extends VlElement {  
-    constructor(driver, selector) {
-        super(driver, selector);
+class VlLink extends VlElement {
+    async getIcons() {
+        const icons = await this.findElements(By.css('[is="vl-icon"]'));
+        return Promise.all(icons.map(icon => new VlIcon(this.driver, icon)));
     }
```

Comment:
er is geen dependency van link naar icon gedefinieerd, dus ofwel moet die er wel zijn, ofwel mogen we hier weer het page-object van VlLink niet koppelen aan dat van VlIcon

Reply:
Inderdaad dan best verantwoordelijkheid bij afnemer leggen die weet wat er in de link zit.

Reply:
Offline afgesproken dat we vl-ui-icon gaan toevoegen als dependency